docs/source/networking/bu-ccds-networks.md

```diff
@@ -0,0 +1,31 @@
+# BU CCDS Networks
+
+2 racks have been allocated to the MOC in the room SB05-A. The racks are labelled 665-PA-F1 and 665-PA-R1.
+
+BU is using Cisco Application Policy Infrastructure Controller (APIC) to manage the switches. There's a tenant called
+"MassOpenCloud" which holds configuration for our racks.
+
+## Public BU (VLAN 122)
+
+This VLAN is enabled on ports 1-4 of both TOR cisco switches.
+
+Cisco APIC Application Profile: `moc-public`
+
+- Subnet: 128.197.248.8/29
+
+```
+IP Address       Hostname/Description
+128.197.8.9      Gateway
```

Comment:
I might be forgetting a subnet rule. would this be 128.197.248.9 and the one below 128.197.248.10?

Reply:
You are correct, that's a mistake. Thanks for catching that!